spaces/S000168/README.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: S000168
+name: Real projective plane
+aliases:
+  - $\mathbb RP^2$
+  - $P_2(\mathbb R)$
```

Comment:
I don't think the alias $P_2(\mathbb R)$ is actually very common. Feel free to disagree.

Reply:
Because I see Wikipedia use it. There is also a notation namely $P(\mathbb R^2)$ as you committed.

Reply:
> I don't think the alias P 2 ( R ) is actually very common. Feel free to disagree. P 2 or P 2 perhaps is though.

$P^2$ (`P^2`) is also very reasonable with only defect not specifying which field is ($\mathbb R$ or $\mathbb C$).

I don't agree $\mathbb P^2$ (`\mathbb P^2`) because I think it is nonsense to wrap $P$ with a blackboard (`\mathbb`).

Reply:
Maybe the notation $\mathbb{P}^2(\mathbb{R})$ occurs in algebraic geometry to refer to the real points of the algbraic variety $\mathbb{P}^2$? I have very weak familiarity with these things though. I tried for a few weeks last summer to learn about linear algebraic groups to varying success...

Reply:
> I don't agree P 2 (`\mathbb P^2`) because I think it is nonsense to wrap P with a blackboard (`\mathbb`).

I don't think it's nonsense. It is used in Lee's Topological manifolds (which I added as a reference). However, I also prefer it without the blackboard bold.